Source/WebCore/html/HTMLAttachmentElement.cpp

```diff
@@ -201,15 +202,18 @@ class AttachmentSaveEventListener final : public EventListener {
     void handleEvent(ScriptExecutionContext&, Event& event) final
     {
         if (event.type() == eventNames().clickEvent) {
+            ASSERT(event.isMouseEvent());
+            auto& mouse = static_cast<MouseEvent&>(event);
+            auto copiedEvent = MouseEvent::create(m_attachment->attributeWithoutSynchronization(saveAttr()), mouse.bubbles() ? Event::CanBubble::Yes : Event::CanBubble::No,
```

Comment:
No need to pass over CanBubble::Yes/No, you should decide whether you want your event to bubble.

Reply:
(by default bubbling is off, so that might be a good default)